Extension/src/LanguageServer/extension.ts

```diff
@@ -812,7 +812,7 @@ function onToggleSquiggles(): void {
     onActivationEvent();
     // This only applies to the active client.
     let settings: CppSettings = new CppSettings(clients.ActiveClient.RootUri);
-    settings.toggleSetting("errorSquiggles", "Enabled", "Disabled");
+    settings.toggleSetting("errorSquiggles", "Disabled", "Enabled");
```

Comment:
Since we have 3 states, we need to decide which 2 we toggle, or if we need to rework this command.  (I don't think toggle will work if they have the other state set)

Reply:
Maybe we just remove the toggle